Automattic/jetpack
# AI Extension: Enable Form extension inside query loops

Fixes #32554

## Proposed changes:

* Enables the Form extension again

It seems that the recent changes made by @retrofox have fixed the issue as a side-effect, so I'm only removing the query loop check and fixing some typescript issues in the same file.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the site editor
* Add a Form inside a query loop
* Check that there is only one AI icon
* Check that the extension works with no regression both on the site editor and in a post

| Before | After
| -|-
|  ![2023-10-18_17-23-36](https://github.com/Automattic/jetpack/assets/8486249/518bf2fa-6460-4507-acf8-e552129a40de) | ![2023-10-18_16-53-44](https://github.com/Automattic/jetpack/assets/8486249/be476732-d58d-4c2d-abb1-96d4dc17bf98)
